Meta/ladybird.sh

```diff
@@ -219,14 +223,10 @@ build_and_run_lagom_target() {
 
     build_target "${lagom_target}"
 
-    if [ "$lagom_target" = "ladybird" ] && [ "$(uname -s)" = "Darwin" ]; then
-        "$BUILD_DIR/bin/Ladybird.app/Contents/MacOS/Ladybird" "${lagom_args[@]}"
+    if [ "$(uname -s)" = "Darwin" ]; then
+        "$BUILD_DIR/bin/Ladybird.app/Contents/MacOS/$lagom_target" "${lagom_args[@]}"
```

Comment:
This is only the correct directory for targets that live in the macOS app bundle. We still build utilities like js and wasm that live in bin/

Reply:
Clearly I should have tested this with something other than `headless-browser` and `WebDriver` :sweat_smile: 

I've updated the PR so that only those targets that are part of the app bundle are changed.

Reply:
This will work, but it's extremely unlikely that starting RequestServer/WebContent/WebWorker/ImageDecoder will work from the command line :man_shrugging: